conformance/tests/gateway-with-attached-routes.go

```diff
@@ -36,6 +36,7 @@ var GatewayWithAttachedRoutes = suite.ConformanceTest{
 	Description: "A Gateway in the gateway-conformance-infra namespace should be attached to routes.",
 	Features: []suite.SupportedFeature{
 		suite.SupportGateway,
+		suite.SupportHTTPRoute,
```

Comment:
@robscott I added this line and L149 based on your comments in https://github.com/kubernetes-sigs/gateway-api/pull/2477#pullrequestreview-1690473146.